Documentation/source/changelog/20180409-25-provide-sys-language-uid.rst

```diff
@@ -0,0 +1,16 @@
+FEATURE 25 "Respect multiple languages" - Provide sys_language_uid
+==================================================================
+
+Previously we did not fetch ``sys_language_uid`` field from database. This prevents everyone from
```

Comment:
`prevents` => `prevented`

We used to prevent the work with multiple languages - not anymore.

Reply:
Thanks, fixed that.